### PR TITLE
make Julia 0.6 compliant

### PR DESCRIPTION
**Pull request by dfdx**

### CODE REVIEW

File: src/schema.jl [hunk 1]
```diff
@@ -92,7 +92,7 @@ function schema_to_proto_schema(io::IO, sch::Vector{SchemaElement}, schema_name:
     end
     lev0 = ios[end]
     println(lev0, "}")
-    write(io, takebuf_string(lev0))
+    write(io, String(take!(lev0)))
```
Comment (tkelman) [inline src/schema.jl]: could you add a Compat 0.9.5 min ver?

Comment (dfdx) [inline src/schema.jl]: Fixed. Zlib and Thrift will also need an updated minimal version when current fixes to these packages (i.e. dcjones/Zlib.jl#32 and tanmaykm/Thrift.jl#28) get merged and released.

Comment (tkelman) [inline src/schema.jl]: not on Julia 0.5 though, don't the existing versions work okay there?

Comment (dfdx) [inline src/schema.jl]: I guess they do. But if we claim that Parquet.jl works on both - Julia 0.5 and 0.6 - shouldn't its `REQURE` file list the minimal version of dependencies that make Parquet.jl to work on both Julia versions?

Comment (tkelman) [inline src/schema.jl]: No, lower bounds shouldn't rule out version combinations that work correctly. You could tag the same sha twice with one set of dependency bounds for `julia 0.5 0.6` and another set of dependency bounds for `julia 0.6`, or upper bound the old versions of the dependencies if they don't even load on julia 0.6. There's no way currently to specify a different set of dependency bounds for different julia versions within the same tag. Pkg3 could be made to support that but Stefan is skeptical of the need.

**Comment by tanmaykm** [comment]
Thanks @dfdx, merging this in a bit.

**Pull request merged**